02_math_operations.py

```diff
@@ -0,0 +1,22 @@
+#------------------------------------------------------------------------
+#
+# Python program to add, subtract, multiple and divide any two numbers
+#
+# AUTHOR  : Anuhya Dasaraju
+# DATE    : 8th March, 2024
+# VERSION : 1.0
+#------------------------------------------------------------------------
+print("Enter 2 numbers to perform arithmetic operations")
+num1 = int(input("num1 : "))
+num2 = int(input("num2 : "))
+
```

Comment:
Input validation is missing.
1) Try giving floating numbers like 10.0 or 12.2 etc
2) Try giving a alphabet instead of number
3) raw_input() - why not use this instead of input() - explore and decide which one to use here.